docs/Developers/Testing_Bears.rst

```diff
@@ -93,15 +93,17 @@ arguments.
 
     ``bad_file`` here is same as ``bad_file`` in the above example.
 
-``check_validity`` asserts if your bear yields any results for a particular
-check with a list of strings. First a *Section* and your Bear
-(in this case ``TooManyLinesBear``) is ``setUp``. Now your *Section* consists
-by default *Settings*. You can append any *Setting* depending on your test.
-Validate a check by passing your bear, lines to check as parameters
-(pass a few other parameters if necessary) to ``check_validity``. The method
-``self.check_validity(self.uut, ["import os"])`` asserts if your bear
-``self.uut`` yields a result when a list of strings ``["import os"]`` is
-passed.
+``check_validity`` asserts if your bear either yields or does not yield any
```

Comment:
>asserts if your bear either yields or does not yield

are you sure about this? if it does both then we can simply use it everywhere :P
I will recommend it to try `check_validity` at least once locally, things get clear :smiley: .
Also, this line is contradicting with line [103](https://github.com/coala/coala/pull/5483/files#diff-10a66d8d7f204ab29daecaf139504eaaR102)

Reply:
I will suggest you make the line 96 to `asserts if your bear does not yield any`